docs/docs/upgrading.md

```diff
@@ -7,6 +7,28 @@ description: >-
 
 # Since 0.15.0
 
+## New
+
+### Policy for Device Identity
+
+This release of Pomerium adds the ability to set policy based on system registration via [WebAuthN](https://www.w3.org/TR/webauthn-2/#registration-extension).
+
+See [device identity](/docs/topics/device-identity.md) for more details.
+
+### HTTP PPL Criteria
+
+`http_path` and `http_method` are now supported for matching HTTP requests in policies. See [PPL](/docs/topics/ppl.md#criteria) for more details.
+
+## Breaking
+
+### Self Signed fallback certificates
+
+When selecting a TLS certificate for a listener, Pomerium attempts to locate one by iterating through the provide certs and searching for a SAN match against a service URL such as `databroker_service_url`.
```

Comment:
1. Would "listener" be analogous to the `from` value in this context?
2. I'm not following how this relates to `databroker_service_url`.

Reply:
Discussed with @desimone, will suggest revised copy in a new PR.